client/blocks/post-share/index.jsx

```diff
@@ -430,13 +445,15 @@ class PostShare extends Component {
 			hasFetchedConnections,
 			hasRepublicizeFeature,
 			hasRepublicizeSchedulingFeature,
+			siteSlug,
+			translate,
 		} = this.props;
 
 		if ( ! hasFetchedConnections ) {
 			return null;
 		}
 
-		const { siteSlug, translate, isJetpack } = this.props;
+		const { isJetpack } = this.props;
```

Comment:
It doesn't matter too much, but just out of curiosity, why not add `isJetpack` to the list above also?